sdk/core/package-lock.json

```diff
@@ -1,46 +1,60 @@
 {
   "name": "@affinidi/wallet-core-sdk",
-  "version": "7.17.1",
+  "version": "7.18.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@affinidi/wallet-core-sdk",
-      "version": "7.17.0",
+      "version": "7.18.0",
       "license": "ISC",
       "dependencies": {
+        "@affinidi/affinidi-did-auth-lib": "^2.9.0",
         "@affinidi/affinity-metrics-lib": "^0.0.25",
+        "@affinidi/common": "^3.1.0",
```

Comment:
handle internal deps with lerna instead of npm. There should not be any internal deops in packge-lock.json. See what were before.
To clean it, ser npm ru clean, npm run bootstrap from root dir

Reply:
Will make the changes.